api/KeyboardEvent.json

```diff
@@ -300,10 +300,10 @@
           "spec_url": "https://w3c.github.io/uievents/#dom-keyboardevent-getmodifierstate",
           "support": {
             "chrome": {
-              "version_added": "31"
+              "version_added": "30"
```

Comment:
Curiously it looks like the Opera data is already correct if mirroring Chromium 30. I'm guessing that was originally due to a mistake, but now made correct.